File: php/URI/3301.php
<?php

$linha = explode(' ', readline());


$h = $linha[0];
$z = $linha[1];
$l = $linha[2];


if (($z > $l && $z < $h) || ($z < $l && $z > $h))
    print("zezinho\n");
else if ($l > $z && $l < $h || $l < $z && $l > $h)
    print("luisinho\n");
else
    print("huguinho\n");

File: C/URI/3301.c
#include<stdio.h>
int main(){

    int h, l, z;

    scanf("%d %d %d", &h, &z, &l);

    if(z > l && z < h || z < l && z > h)
        printf("zezinho\n");
    else if(l > z && l < h || l < z && l > h)
        printf("luisinho\n");
    else   
        printf("huguinho\n");
    

    return 0;
}